src/trace_processor/importers/ftrace/ftrace_parser.cc

```diff
@@ -2611,17 +2629,87 @@ void FtraceParser::ParseCmaAllocStart(int64_t timestamp, uint32_t pid) {
   UniqueTid utid = context_->process_tracker->GetOrCreateThread(pid);
   TrackId track_id = context_->track_tracker->InternThreadTrack(utid);
 
+  // clear migration info for the new allocation sequence.
```

Comment:
Is there any reason to do this erase? Are you ever going to get `CmaAllocStart ` without a `MmAllocContigMigrateRangeInfo` to clear it out?

Doing an erase unnecesary will kill performance for little reason. Having a bit of memory around is not the end of the world, hurting perf on every CMA event is much worse.

Reply:
Also if you don't need erase, you should definitely switch to base::FlatHashMap. It's much better for cases like that.